File: StreamingApp.Domain/DTOs/SongDto.cs
using StreamingApp.Shared.Enums;
using System;

namespace StreamingApp.Domain.DTOs
{
    public class SongDto
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Url { get; set; }

        public Genre Genre { get; set; }

        public int Playbacks { get; set; }

        public DateTime AddedOn { get; set; }

        public int Likes { get; set; }

        public ApplicationUserDto Author { get; set; }
    }
}

File: StreamingApp/ClientApp/src/app/api/dtos/song-dto.ts
import { Genre } from "src/app/helpers/genre.enum";
import { ApplicationUserDto } from "./application-user-dto";

export interface SongDto {
    id: number;
    name: string;
    url: string;
    genre: Genre;
    playbacks: number;
    likes: number;
    likedByUser: boolean;
    addedOn: Date;
    author: ApplicationUserDto;
};